routers/user/profile.go

```diff
@@ -22,7 +22,6 @@ import (
 
 const (
 	tplFollowers base.TplName = "user/meta/followers"
-	tplStars     base.TplName = "user/meta/stars"
```

Comment:
The related file should be remove `/templates/user/meta/stars.tmpl`

Reply:
That file was actually empty, thanks!